agent/checks/check_test.go

```diff
@@ -44,7 +43,7 @@ func TestCheckMonitor_Script(t *testing.T) {
 	for _, tt := range tests {
 		t.Run(tt.status, func(t *testing.T) {
 			notif := mock.NewNotify()
-			logger := log.New(ioutil.Discard, uniqueID(), log.LstdFlags)
+			logger := testutil.TestHcLog("")
```

Comment:
These could be improved if they used a hclogger that ultimately fed logs into a `testutil.TestWriter` so that logs were correlated with test cases.

Reply:
Unless the logs are meaningless in the context of the test in which case something like the aforementioned to-be-made `testutil.NewDiscardLogger` could go here instead.